packages/ui/src/domains/LandingPageBuilder/LandingPageBuilder.tsx

```
@@ -1505,68 +1593,86 @@
+						onChange={(e) => onUpdate({ src: e.target.value })}
+						placeholder="Video URL"
+						className="mb-2"
 					/>
-					<Label htmlFor="controls">Show Controls</Label>
-				</div>
-			</div>
+					<div className="flex flex-wrap gap-2">
+						<Input
+							type="number"
+							value={width}
+							onChange={(e) => onUpdate({ width: parseInt(e.target.value) })}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)